devices/pine64-dont-be-evil/default.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+
+{
+  mobile.device.name = "pine64-dont-be-evil";
+  mobile.device.info = rec {
+    format_version = "0";
+    name = "PinePhone Don't be evil development kit";
+    manufacturer = "PINE64";
+    date = "";
+    modules_initfs = "";
+    arch = "aarch64";
+
+    keyboard = false;
+    external_storage = true;
+
+    screen_width = "720";
+    screen_height = "1440";
+
+    flash_method = "fastboot";
+
+    kernel_cmdline = "console=tty0 console=ttyS0,115200 no_console_suspend earlycon=uart,mmio32,0x01c28000 panic=10 consoleblank=0 loglevel=8";
+
+    # Hmm, this smells fishy, android bootimg in an allwinner device?
+    # Read on!
+    generate_bootimg = true;
+
+    # This is the usual for allwinner.
+    flash_offset_base    = "0x40000000";
+    flash_offset_kernel  = "0x00080000";
+    flash_offset_ramdisk = "0x0fe00000";
+    # The next two don't really matter I think
+    flash_offset_second  = "0x00000000";
+    flash_offset_tags    = "0x00000100";
+    flash_pagesize = "2048";
+
+    # FIXME: Use a more appropriate option for appending the dtb to the boot.img.
+    bootimg_qcdt = true;
+
+    # TODO : make kernel part of options.
+    kernel = pkgs.callPackage ./kernel {
+      kernelPatches = [];
+      #kernelPatches = pkgs.defaultKernelPatches;
```

Comment:
Remove this

(Additionally review the whole file compared to a recent device addition.)